sql/mysqld.cc

```diff
@@ -5033,6 +5033,21 @@ a file name for --log-bin-index option", opt_binlog_index_name);
   }
   plugins_are_initialized= TRUE;  /* Don't separate from init function */
 
+  LEX_CSTRING keyring_vault_name= {C_STRING_WITH_LEN("keyring_vault")};
+  LEX_CSTRING keyring_name= {C_STRING_WITH_LEN("keyring_file")};
+  if ((plugin_is_ready(keyring_vault_name, MYSQL_KEYRING_PLUGIN) ||
+       plugin_is_ready(keyring_name, MYSQL_KEYRING_PLUGIN)) &&
+      !pxc_encrypt_cluster_traffic)
```

Comment:
This is only checking pxc_encrypt_cluster_traffic and does not check for SST encryption (as indicated by the comment).  So this check would fail if only SST encryption was setup (with pxc_encrypt_cluster_traffic = OFF).

Reply:
@kennt-percona - mysqld binary only reads mysqld section of my.cnf files. This fix was intended to keep that way. 
Are you suggesting we should change the logic and make it read [sst] section as well? If so this seems rather a big change as we will need to make sure it only reads it on a temporary buffer/variable and do not start the server with those options

Reply:
Also, the user can change my.cnf adding [sst] section after mysqld has started in order to make SST works. So I'm not sure how valid is to implement a check to read [sst] section of my.cnf at startup since it can be changed dynamically.

Reply:
The issue is that we are telling the user how to fix it, but if they do it that way, the warning will still appear.  Sounds like a recipe for a frustrated user (we're saying to the user, there are two ways to fix the issue, but just one way will remove the warning).
It may be better to just remove the portion about enabling it via the [sst] section.

Reply:
@kennt-percona ack, fixed

Reply:
lgtm